### PR TITLE
Bump version to 0.7.0

### PR DESCRIPTION
**Pull request by NivekT**
Stack from [ghstack](https://github.com/ezyang/ghstack):
* __->__ #1028

Differential Revision: [D43370795](https://our.internmc.facebook.com/intern/diff/D43370795)

### CODE REVIEW

**Comment by NivekT** [comment]
@nivekt has imported this pull request.  If you are a Meta employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D43370795).

**Review by ejguan — APPROVED** [review]
LGTM, thanks

**Comment by ejguan** [comment]
One sec,pls not land it.

File: README.md [hunk 1]
```diff
@@ -53,7 +53,8 @@ The following is the corresponding `torchdata` versions and supported Python ver
 
 | `torch`              | `torchdata`        | `python`          |
 | -------------------- | ------------------ | ----------------- |
-| `master` / `nightly` | `main` / `nightly` | `>=3.8`, `<=3.10` |
+| `master` / `nightly` | `main` / `nightly` | `>=3.8`, `<=3.11` |
+| `2.0.0`              | `0.6.0`            | `>=3.8`, `<=3.11` |
```
Comment (ejguan) [inline README.md:57]: Ohh, I am sorry. We should not touch compatibility table until the official release is provided. My bad. Let's revert the last commit

Comment (NivekT) [reply README.md:57]: Reverting

Comment (NivekT) [reply README.md:57]: I think nightly should still be `<=3.11`? What do you think?

Comment (ejguan) [reply README.md]: Yeah

**Review by ejguan — APPROVED** [review]
Thank you

**Comment by NivekT** [comment]
@nivekt has imported this pull request.  If you are a Meta employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D43370795).

**Pull request closed without merging**